static/js/pages/instances/InstanceList.tsx

```diff
@@ -186,23 +185,13 @@ const InstanceList: FC = () => {
           </>
         }
         controls={
-          <ContextualMenu
-            hasToggleIcon
-            links={[
-              {
-                children: "Quick create instance",
-                onClick: () => panelParams.openInstanceFormGuided(),
-              },
-              {
-                children: "Custom create instance (YAML)",
-                onClick: () => panelParams.openInstanceFormYaml(),
```

Comment:
Yes, I was already taking that into account, but since the other YAML-based instance creation form is also used to edit instances, I think it would be better to update it so that it handles the edit case only, instead of deleting it completely. So I created a dedicated ticket for this ([WD-1358](https://warthogs.atlassian.net/browse/WD-1358)), feel free to add details there about this point in case I forgot anything. I plan to work on that right after this PR gets merged, but for what I have just mentioned I would prefer to consider it a separate issue and out of the scope of this PR.

Reply:
sounds good, thanks.